deploy/materialized_view_fuel.sql

```diff
@@ -0,0 +1,61 @@
+             substances_idx integer path 'string(count(./ancestor::Substance/parent::*/preceding-sibling::*))' not null,
+             substance_idx integer path 'string(count(./ancestor::Substance/preceding-sibling::Substance))' not null,
+             fuel_idx integer path 'string(count(./preceding-sibling::Fuel))' not null,
+             fuel_type varchar(1000) path './FuelType',
+             fuel_classification varchar(1000) path './FuelClassification',
+             fuel_units varchar(1000) path './FuelUnits',
+             annual_fuel_amount varchar(1000) path './AnnualFuelAmount',
+             annual_weighted_avg_hhv varchar(1000) path './AnnualWeightedAverageHighHeatingValue'
+         ) as fuel_details
+) with no data;
+
+create unique index ggircs_fuel_primary_key on ggircs_swrs.fuel (ghgr_import_id, activity_name, sub_activity_name, unit_name, sub_unit_name, process_idx, sub_process_idx, units_idx, unit_idx, substances_idx, substance_idx, fuel_idx);
+
+comment on materialized view ggircs_swrs.fuel is 'The materialized view containing the information on fuels';
+comment on column ggircs_swrs.fuel.ghgr_import_id is 'A foreign key reference to ggircs_swrs.ghgr_import';
+comment on column ggircs_swrs.fuel.activity_name is 'The name of the activity (partial fk reference)';
+comment on column ggircs_swrs.fuel.sub_activity_name is 'The name of the sub_activity (partial fk reference)';
+comment on column ggircs_swrs.fuel.unit_name is 'The name of the unit (partial fk reference)';
+comment on column ggircs_swrs.fuel.sub_unit_name is 'The name of the sub_unit (partial fk reference)';
+comment on column ggircs_swrs.fuel.process_idx is 'The number of preceding Process siblings before this Process (partial fk reference)';
+comment on column ggircs_swrs.fuel.sub_process_idx is 'The number of preceding SubProcess siblings before this SubProcess (partial fk reference)';
+comment on column ggircs_swrs.fuel.units_idx is 'The number of preceding Units siblings before this Units (partial fk reference)';
+comment on column ggircs_swrs.fuel.unit_idx is 'A foreign key reference to ggircs_swrs.unit (partial fk reference)';
+comment on column ggircs_swrs.fuel.substances_idx is 'The number of preceding siblings before the Substances';
+comment on column ggircs_swrs.fuel.substance_idx is 'The number of preceding siblings before the Substance';
+comment on column ggircs_swrs.fuel.fuel_idx is 'The primary key for the fuel';
+comment on column ggircs_swrs.fuel.fuel_type is 'The type of the fuel';
+comment on column ggircs_swrs.fuel.fuel_classification is 'The classification of the fuel';
+comment on column ggircs_swrs.fuel.fuel_units is 'The units of the fuel';
+comment on column ggircs_swrs.fuel.annual_fuel_amount is 'The annual amount of the fuel';
+comment on column ggircs_swrs.fuel.annual_weighted_avg_hhv is 'The annual weight avg of the high heating value of the fuel';
+
```

Comment:
@wenzowski Do these comments, or even the names need a re-think since what is being pulled may not be what is described? ie Since we are pulling parents of ancestors or parents of parents of ancestors and allowing them to be named anything (*) do the current comments and names still make sense?

Reply:
I think we're going to have to do a full review of our metadata. Maybe we should create a ticket to note that we've got tech-debt in our comments.